tokio-buf/src/ext/bytes.rs

```diff
@@ -1,5 +1,5 @@
 use BufStream;
-use buf_stream::errors::internal::Never;
+use errors::internal::Never;
```

Comment:
I believe this file probably wants to be moved outside of the ext module since its not really an extension but just some blanket impls.